notebooks/book2/17/gprDemoChangeHparams.ipynb

```diff
@@ -2,14 +2,152 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def eucledian_metric(a, b):
</pre><p>This function should perhaps just use <code>jax.vmap</code> or simple broadcasting instead of nested loops. Also, the spelling of Euclidean is wrong.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>